src/core_plugins/input_control_vis/public/control/filter_manager/phrase_filter_manager.js

```diff
@@ -43,14 +41,25 @@ export class PhraseFilterManager extends FilterManager {
     if (kbnFilters.length === 0) {
       return this.getUnsetValue();
     } else {
-      const values = kbnFilters
+      return kbnFilters
```

Comment:
Your reduce seems to be just doing a clone of the array. If I'm reading this correctly, this entire statement could become this:

```js
return kbnFilters.map((kbnFilter) => {
  const value = this._getValueFromFilter(kbnFilter);
  return { value, label: value };
});

```

Reply:
The reduce is joining any array of arrays into a single array.

Reply:
Ah. I had missed that. Seems `_.flatten` might be used, then.

Reply:
I would prefer to leave this the way it is. Native javascript syntax is understandable in this case